dockstore-webservice/src/main/java/io/dockstore/webservice/core/Container.java

```diff
@@ -221,8 +221,8 @@ public String getPath() {
             StringBuilder builder = new StringBuilder();
             if (registry == Registry.QUAY_IO) {
                 builder.append("quay.io/");
-            } else {
-                builder.append("registry.hub.docker.com/");
+                // } else {
```

Comment:
Was this deleted on purpose? In my testing, this was required so that the refresh would function properly with images from docker hub.